.scripts/util.sh

```diff
@@ -59,25 +63,73 @@ is_generated() {
     fi
 }
 
+select_browser() {
```

Comment:
This one should probably be `select_browser_caller` or suchlike as it is not at the top level API.